styles/style.css

```diff
@@ -372,3 +377,232 @@ hr {
   background-color: #2230d2;
   border-color: #2230d2;
 }
+
+@media screen and (min-width: 992px) {
+  .header {
+    align-items: baseline;
+  }
+
+  .logo {
+    margin: 25px 155px;
+  }
+
```

Comment:
- [ ] **OPTIONAL**
For changing the look of the website on the medium-sized screen you could use another media query this way
```
@media screen and (min-width: 365px) and (max-width: 991px) {
//
}
```
![Screenshot](https://user-images.githubusercontent.com/47625407/121396581-d05a9d00-c953-11eb-879c-f05eb779e941.png)